# https://github.com/solidjs/solid-start/discussions/1403
diff --git a/dist/router/lazyRoute.js b/dist/router/lazyRoute.js
index 463636a16572208dfbd801e758dc45c0657ddccc..0423d57677b84f5fbc0f8d9d06e0c8918d35f709 100644
--- a/dist/router/lazyRoute.js
+++ b/dist/router/lazyRoute.js
@@ -37,7 +37,7 @@ export default function lazyRoute(component, clientManifest, serverManifest, exp
             return { default: Comp };
         }
         else {
-            const mod = await component.import();
+            const mod = await component.build();
             const Component = mod[exported];
             let assets = await clientManifest.inputs?.[component.src].assets();
             const styles = assets.filter((asset) => asset.tag === "style" || asset.attrs.rel === "stylesheet");
